src/dependabot/update_metadata.ts

```diff
@@ -38,8 +38,8 @@ export async function parse (commitMessage: string, branchName: string, mainBran
     // Since we are on the `dependabot` branch (9 letters), the 10th letter in the branch name is the delimiter
     const delim = branchName[10]
     const chunks = branchName.split(delim)
-    const prev = bumpFragment?.groups?.from ?? ''
-    const next = bumpFragment?.groups?.to ?? ''
+    const prev = bumpFragment?.groups?.from.replace('v', '') ?? ''
+    const next = bumpFragment?.groups?.to.replace('v', '') ?? ''
```

Comment:
I think if you move this `replace()` from here to where the update is actually calculated then it'll preserve the `"v"`